observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,6 +12,143 @@ You can use the {TempoName} xref:../otel/otel-forwarding-telemetry-data.adoc#ote
+
+////
+[id="distr-tracing_3-3-1_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-202?-????/[CVE-202?-????]
+////
+
+[id="distr-tracing_3-3-1_tempo-release-notes_{context}"]
+=== {TempoName}
+
+The {TempoName} is provided through the {TempoOperator}.
+
+The {TempoName} 3.3.1 is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.5.0.
+
+////
+[id="distr-tracing_3-3-1_tempo-release-notes_technology-preview-features_{context}"]
+==== Technology Preview features
+
+This update introduces the following Technology Preview feature:
+
+* ???.
+
+:FeatureName: The Tempo monolithic deployment
+include::snippets/technology-preview.adoc[leveloffset=+1]
+////
+
+////
+[id="distr-tracing_3-3-1_tempo-release-notes_new-features-and-enhancements_{context}"]
+==== New features and enhancements
+
+This update introduces the following enhancements:
+
+* ???.
+////
+
+////
+[id="distr-tracing_3-3-1_tempo-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+In the {TempoName} 3.3.1, ???.
+////
+
+////
+[id="distr-tracing_3-3-1_tempo-release-notes_removal-notice_{context}"]
+==== Removal notice
+
+In the {TempoName} 3.3.1, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+////
+
+////
+[id="distr-tracing_3-3-1_tempo-release-notes_bug-fixes_{context}"]
+==== Bug fixes
+
+This update introduces the following bug fixes:
+
+* ????. (link:https://issues.redhat.com/browse/TRACING-????[TRACING-????])
+////
+
+[id="distr-tracing_3-3-1_tempo-release-notes_known-issues_{context}"]
+==== Known issues
+
+There is currently a known issue:
+
+* Currently, the {TempoShortName} fails on the {ibm-z-title} (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+
+[id="distr-tracing_3-3-1_jaeger-release-notes_{context}"]
+=== {JaegerName}
+
+The {JaegerName} is provided through the {JaegerOperator} Operator.
+
+The {JaegerName} 3.3.1 is based on the open source link:https://www.jaegertracing.io/[Jaeger] release 1.57.0.
+
+[IMPORTANT]
+====
+Jaeger does not use FIPS validated cryptographic modules.
+====
+
+[id="distr-tracing_3-3-1_jaeger-release-notes_support-for-elasticsearch-operator_{context}"]
+==== Support for the {es-op}
+
+The {JaegerName} 3.3.1 is supported for use with the {es-op} 5.6, 5.7, and 5.8.
+
+[id="distr-tracing_3-3-1_jaeger-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+In the {DTProductName} 3.3.1, Jaeger and support for Elasticsearch remain deprecated, and both are planned to be removed in a future release.
+Red Hat will provide support for these components and fixes for CVEs and bugs with critical and higher severity during the current release lifecycle, but these components will no longer receive feature enhancements.
```

Comment:
```suggestion
Red{nbsp}Hat will provide support for these components and fixes for CVEs and bugs with critical and higher severity during the current release lifecycle, but these components will no longer receive feature enhancements.
```
See https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces.

Reply:
Thank you so much. I fixed this throughout the file.